Apps/WebClient/src/ClientApp/src/store/modules/credential/actions.ts

```diff
@@ -44,6 +44,28 @@ export const actions: CredentialActions = {
                 });
         });
     },
+    disconnectConnection(
+        context,
+        params: { hdid: string; connectionId: string }
+    ): Promise<boolean> {
+        const credentialService: ICredentialService =
+            container.get<ICredentialService>(
+                SERVICE_IDENTIFIER.CredentialService
+            );
+        return new Promise((resolve, reject) => {
+            context.commit("setRequested");
+            credentialService
+                .disconnectConnection(params.hdid, params.connectionId)
+                .then((result) => {
+                    context.commit("removeCredential", result);
```

Comment:
Shouldn't we add a specific mutation for removing the connection?  I don't see how `removeCredential` would handle that.

Reply:
good catch